typescript-final/package.json

```
@@ -10,7 +10,7 @@
   "dependencies": {
     "date-fns": "^2.11.1",
     "gray-matter": "^4.0.2",
-    "next": "^10.0.0",
+    "next": "^11.0.0",
     "react": "16.13.1",
     "react-dom": "16.13.1",
```

```suggestion
    "react": "17.0.2",
    "react-dom": "17.0.2",
```

Do we want to update these too? Also, wasn't there a specific TypeScript version we need to be above?